doc/sphinx/source/build/project_build.rst

```diff
@@ -143,7 +137,7 @@ and generate executable file (.bin/.hex)
                .. image:: /source/build/stm32_exclude_files_option.png
                   :width: 600
 
-            * In order to choose STM32 platform in the firmware, select the "ACTIVE_PLATFORM" as "STM32_PLATFORM" in the app_config.h from the respective project.
+            * In order to choose STM32 platform in the firmware, select the "ACTIVE_PLATFORM" as "STM32_PLATFORM" in the app_config.h from the respective project. Alternately , add compiler flag "ACTIVE_PLATFORM == value of STM32_PLATFORM in app_config.h" for selecting stm32 platform.
```

Comment:
"==" is not needed. It just has to be ACTIVE_PLATFORM=2
I am also not sure if space characters are accepted. I don't usually add spaces to these flags